core/src/main/java/org/infinispan/configuration/cache/CacheMode.java

```diff
@@ -5,6 +5,11 @@
  */
 public enum CacheMode {
    /**
+    * Data is not replicated and certain features are not possible
```

Comment:
The cache cannot be injected transparently, and from configuration perspective there would be many prohibited configurations when you'd set the simple-cache="true" attribute. As this is not an orthogonal configuration option, I've decided to have that as new cache mode.
Can you support your opinion with any arguments we could discuss?